controllers/servingruntime_controller.go

```diff
@@ -286,6 +298,60 @@ func (r *ServingRuntimeReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	return ctrl.Result{RequeueAfter: requeueDuration}, nil
 }
 
+func (r *ServingRuntimeReconciler) getPVCs(ctx context.Context, req ctrl.Request, rt *kserveapi.ServingRuntimeSpec, cfg *config.Config) ([]string, error) {
+	s := &corev1.Secret{}
+	if err := r.Client.Get(ctx, types.NamespacedName{
+		Name:      modelmesh.StorageSecretName,
+		Namespace: req.Namespace,
+	}, s); err != nil {
+		return nil, fmt.Errorf("Could not get the storage secret: %w", err)
+	}
```

Comment:
To use PVC, why do we need to check StorageSecretName The default "storage-config" secret usually has credentials for accessing cloud storage such as S3. However, PVC does not need credentials to attach to the pod.
With this code block, the controller keeps checking if the secret exists in a namespace. So although it is not needed, it must exist in the namespace to deploy an inference pod.

Reply:
This goes back to some of the design decisions for the implementation, i.e. ...

https://github.com/kserve/modelmesh-serving/issues/230#issuecomment-1260136468

> The controller will watch the storage config secret in each namespace and keep the pvcs mounted to runtime deployments based on any pvc type entries
Predictors can always reference one of these PVCs for storage

https://github.com/kserve/modelmesh-serving/issues/230#issuecomment-1277980357

> The entry type will be checked and the name used to create volumes and volume mounts for runtime deployments, while the keys are irrelevant.

Reply:
so even though users want to use PVC only, creating `storage-config` Secret with dummy data is required, am I right?

Reply:
If you use the `allowAnyPVC` config parameter, and don't want to deploy the minio-examples image, I suppose you would not need it 
Certainly it should not be required to create it with dummy data, so I think we should tolerate it not being there. @njhill

Reply:
@ckadner yes as discussed offline we should change it so that in that mode the storage-config secret isn't required.